### PR TITLE
deploy script updates for curve v2

### PR DESCRIPTION
**Pull request by carlomazzaferro**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/sdk/chainDeploy/mainnets/bsc.ts [hunk 1]
```diff
@@ -300,6 +294,13 @@ const curveV2Pools: CurveV2PoolConfig[] = [
     // eps BUSD jCHF
     lpToken: underlying(assets, assetSymbols["JCHF-BUSD"]),
     pool: "0xBcA6E25937B0F7E0FD8130076b6B218F595E32e2",
+    baseToken: underlying(assets, assetSymbols.BUSD),
```
Comment (vminkov) [inline packages/sdk/chainDeploy/mainnets/bsc.ts:297]: is it safe enough to configure the base token manually in the config? is there an on-chain call that can be used to set the base token in the oracle initialization?

File: packages/sdk/chainDeploy/helpers/oracles/curveLp.ts [hunk 2]
```diff
@@ -79,7 +78,7 @@ export const deployCurveV2LpOracle = async ({
       execute: {
         init: {
           methodName: "initialize",
-          args: [[], [], deployConfig.stableToken, mpo.address],
+          args: [[], [], []],
         },
       },
```
Comment (vminkov) [inline packages/sdk/chainDeploy/helpers/oracles/curveLp.ts:83]: add the `reinitialize` call here

File: packages/sdk/chainDeploy/mainnets/bsc.ts (hunk 1 above)
Comment (carlomazzaferro) [reply packages/sdk/chainDeploy/mainnets/bsc.ts:297]: I am trying to find that out in the open discord for Curve. So far, nothing concrete. Will dig the code.

File: packages/sdk/chainDeploy/helpers/oracles/curveLp.ts (hunk 2 above)
Comment (carlomazzaferro) [reply packages/sdk/chainDeploy/helpers/oracles/curveLp.ts:83]: Will resolve as this will be redeployed

**Pull request merged**